### PR TITLE
Added Docs Build Guide

### PR DESCRIPTION
**Pull request by pokurt**
hope this helps:

I successfully built source using my own guide in a `vnenv`. I thought i might share this here: https://nekobin.com/xerumuwaze

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
-sphinx
-sphinx_rtd_theme
-sphinx_copybutton
-sphinx_tabs
-pypandoc
-requests
-sphinx-autobuild
\ No newline at end of file
+Sphinx==3.3.1
```
Comment (delivrance) [inline docs/requirements.txt:1]: I don't think I'd care much about strict versioning here. I'd rather always use the latest available and if something breaks (rarely, if ever) we fix it in the code. It's not a critical part and is far less annoying than maintaining exact versions.

File: docs/README.md [hunk 2]
```diff
@@ -5,4 +5,17 @@
...
+- Install dependencies: `pip install -r requirements.txt && pip install tgcrypto`
+- Install pyrogram from source: `python setup.py install --user`
+- Change Directory to `docs` and install depenedencies: `cd docs && pip install -r requirements.txt`
+- Run releases inside `scripts` directory: `cd scripts && python releases.py`
+- Change back to project root and run `cd .. && cd .. && python setup.py generate --docs`
+- Finally go to `docs` directory and run `make html`: `cd docs && make html`
+
+### Note
+- Both builds will take a few minutes depending on your machine
+- Ignore the Warning: `WARNING: html_static_path entry '_static' does not exist`
```
Comment (delivrance) [inline docs/README.md:20]: Not sure about this, I can't remember getting it

File: docs/README.md [hunk 3]
```diff
@@ -5,4 +5,17 @@
...
\ No newline at end of file
+## Build Guide
+- make a venv: `python3 -m venv virtualenv`
+- active virtual environment: `source virtualenv/bin/activate`
+- Install dependencies: `pip install -r requirements.txt && pip install tgcrypto`
+- Install pyrogram from source: `python setup.py install --user`
+- Change Directory to `docs` and install depenedencies: `cd docs && pip install -r requirements.txt`
+- Run releases inside `scripts` directory: `cd scripts && python releases.py`
+- Change back to project root and run `cd .. && cd .. && python setup.py generate --docs`
+- Finally go to `docs` directory and run `make html`: `cd docs && make html`
```
Comment (delivrance) [inline docs/README.md:16]: all of this can be simplified. also, I've added a live-html build (with `make lhtml`) a while ago, it could be mentioned here.

Comment (pokurt) [reply docs/README.md:16]: Oh im currently using that i found in `Makefile`. will push new changes tomorrow, no worries

File: docs/requirements.txt (hunk 1 above)
Comment (pokurt) [reply docs/requirements.txt:1]: I had a few problems with python3.7 installing old Sphinx v1. that is why ive done strict versioning here in `requirements.txt`

but i could change that if you want

File: docs/README.md (hunk 2 above)
Comment (pokurt) [reply docs/README.md:20]: me and @GodSaveTheDoge got it for some reason he told me ignore it.

File: docs/README.md (hunk 3 above)
Comment (pokurt) [reply docs/README.md]: I have resolved it by editing the last line. works well!

File: docs/README.md (hunk 2 above)
Comment (GodSaveTheDoge) [reply docs/README.md:20]: Yes, I got it too and the output looked fine so I just ignored it ¯\_(ツ)_/¯

File: docs/requirements.txt (hunk 1 above)
Comment (ColinShark) [reply docs/requirements.txt:1]: Late to the party, but you can also use comparison operators to get not just one specific version, but just "newer than 3.0": `Sphinx>=3.0.0`

**Pull request closed without merging**